magento/magento2
# Magento#25894: fixed issue when "insert image window" didn't refresh after removing folder

### Description (*)

I am only fixed this issue.
I am continuing with tests

### Fixed Issues (if relevant)

1. Fixes magento/magento2#25894: The added image does not disappear from "Insert File" window if the image containing folder has been Deleted

### Manual testing scenarios (*)

1. Create CMS Page or Edit CMS Page
2. Click "Show / Hide Editor" to disable WYSIWYG
3. Click "Insert Image..." to open "Insert File Modal window"
4. Create new folder
5. Select created folder
6. Add file to created folder
7. Remove created folder

"Upload Images" should be refreshed, file(s) should be removed form server.

### Questions or comments


### Contribution checklist (*)
